builtin/types/chainconfig/chainconfig.proto

```diff
@@ -0,0 +1,95 @@
+syntax = "proto3";
+package chainconfig;
+import "github.com/loomnetwork/go-loom/types/types.proto";
+
+message InitRequest {
+    Address owner = 1;
+    Params params = 2;
+}
+
+message GetConfigRequest {
+    string name = 1;
+}
+message GetFeatureRequest {
+    string name = 1;
+}
+
+message AddFeatureRequest {
+    string name = 1;
+}
+
+message AddFeatureResponse {
+}
+
+message GetFeatureResponse {
+    FeatureInfo featureInfo = 1;
+}
+
+message FeatureInfo {
+    Feature feature = 1;
+    uint64 percentage = 2;
+}
+
+message Config {
+    string name = 1;
+    string value = 2;
+}
+
+message Feature {
+    string name = 1;
+    enum FeatureStatus {
+        PENDING = 0;
+        WAITING = 1;
+        ENABLED = 2;
+        DISABLED = 3;
+    }
+    FeatureStatus status = 2;
+    repeated Address validators = 3;
+    uint64 blockHeight = 4;
+}
+
+message Params {
+    uint64 voteThreshold = 1;
+    uint64 numBlockConfirmations = 2;
+}
+
+message SetParamsRequest {
+    Params params = 1;
+}
+
+message GetParamsRequest {
+}
+
+message GetParamsResponse {
+    Params params = 1;
+}
+
+message ListConfig {
+}
+
+message ListFeaturesRequest {
+}
+
+message ListFeaturesResponse {
+    repeated FeatureInfo featureInfos = 1;
```

Comment:
-> `features`

Reply:
We have both `Feature` and `FeatureInfo`. Won't this cause type confusion?